extensions/mssql/package.json

```diff
@@ -190,24 +190,6 @@
           "light": "resources/light/groupBySchemaDisabled.svg"
         }
       },
-      {
-        "command": "mssql.enableGroupBySchemaTitle",
```

Comment:
So we're ok with the title bar tooltip just saying "Group by Schema" without any context that it's specifically for MSSQL?

@aasimkhan30 @erinstellato-ms 

Current tooltip:

![image](https://github.com/microsoft/azuredatastudio/assets/28519865/ef104092-d3c4-48ac-85ba-6b10af576c31)

Reply:
I didn't know the context of Hover text, addressing it.

Reply:
Yeah, sorry - I just remembered that's why we had all this in the first place >.<